src/tests/ftest/util/agent_utils.py

```diff
@@ -32,27 +32,123 @@
 import errno
 import fcntl
 import re
+import yaml
```

Comment:
(pylint-import-error) Unable to import 'yaml'

Reply:
Issue should be revolved after commited of PR-1415.

Reply:
Issue should be resolved after commited of PR-1415